packages/falso/src/lib/text.ts

```diff
@@ -1,6 +1,37 @@
 import { rand } from './core';
-import { data } from './text.json';
+import { word } from './word';
 
-export function text() {
-  return rand(data);
+function randomSentence() {
+  return rand([
+    `${word()} ${word()} ${word()}.`,
+    `${word()} ${word()} ${word()} ${word()}.`,
+    `${word()} ${word()} ${word()} ${word()} ${word()}.`,
+    `${word()} ${word()} ${word()} ${word()} ${word()} ${word()}.`,
```

Comment:
I think we can use JS to do it for us using loops.

Reply:
Even with the punctuation?

Reply:
Be creative :)

Reply:
Ok I think I have come up with a better solution. I have also had to update the `word` function to allow the first letter to be capitalised. I hope this is ok?